summingbird-core-test/src/main/scala/com/twitter/summingbird/MemoryArbitraries.scala

```diff
@@ -0,0 +1,23 @@
+import com.twitter.summingbird.{ KeyedProducer, Producer }
```

Comment:
This is missing new line at end of file, package statement at the top and the apache header